### DISCUSSION TITLE
how to proceed when PPG proposals are related to proposals before the IBC nomenclature committee

### DISCUSSION THREAD

**Discussion by crothfels** [pteridogroup/ppg#45] [General]
Thank you Ashley (and Ivan), for raising this important issue: what to do when a proposed change to PPG I relies upon (or is at least impacted by) a formal nomenclatural proposal that has not yet been ruled upon by the IBC nomenclature committee --this is a situation we didn't think through thoroughly in advance. 

Eric, Joel, and I had a conversation about it at our last organization meeting, however, and would like to suggest that we (the PPG community) consider proposals that involve future decisions by the nomenclature committee provided that:
1. The formal nomenclatural proposal (for conservation, etc) be already made, and 
2. The PPG proposal be explicit about the relationship between the proposed change to PPG I and the nomenclatural proposal. For example, Libing provided an excellent breakdown of the consequences of acceptance/rejection of the proposal to recognize multiple genera within Selaginellaceae in the context of the acceptance/rejectin of the proposal to conserve Selaginella with a new type. 

If we adhere to these two conditions, PPG members can evaluate the PPG proposal fully, including their own assessment of the likelihood of success of the nomenclatural proposal. 

The alternative--not accepting proposals that are contingent on future decisions of the nomenclature committee--could put us in the awkward position of having to make a bunch of nomenclatural acts ourselves, acts that are quite likely to be rendered unnecessary once the nomenclatural proposals are formally ruled upon. The Dennstaedtioideae case provides a good example (see issues #4 and #6 ). Here, recent phylogenetic analyses demonstate that there are four major clades in Dennstaedtioideae: one the comprises Microlepia (but also includes the type of Dennstaedtia), one that comprises most of the species historically recognized in Dennstaedtia, and two smaller clades. To avoid having to rename the members of the Microlepia clade under Dennstaedtia, and move most of the current Dennstaedtia species to a new genus (Patania), the authors published a proposal to conserve Dennstaedtia with a new type (one from a member of the large Dennstaedtia clade). That way Microlepia can stay Microlepia, and Dennstaedtia can stay Dennstaedtia, with only a few species changing genus. However, if we (PPG) disallow proposals that are contingent on decisions by the nomenclatural committee, we would need to transfer all those Microlepia to Dennstaedtia, and all those Dennstaedtia to Patania, since we require that our generic concepts be monophyletic when possible.

Also, two features make this overall issue perhaps less important than it might seem. First, we're going to adopt a more flexible approach to PPG in the future, so we can always (pending a vote) adopt a different classification subsequently, if a nomenclatural proposal is rejected. Second, this situation won't come up very often--the Dennstaedtia and Selaginella proposals are the only proposals before the nomenclature committee that we know of that have the potential to impact fern classification at the level of genus or above.

As always, we welcome discussion and disagreement, so feel free to propose other approaches if you think this one is unsatisfactory.

cheers,
Carl

**Comment by DrAshleyField** [comment]
Thanks Carl,

How does our PPG II timeline (it is September 2023 now) compare to the IBC timeline (July 2024)?
Also, would it be possible to include a provisional vote, e.g.:

**Yes** (regardless of outcome of the nomenclature proposal)
**Provisional Yes** (pending the acceptance of the nomenclature proposal)
**No** (regardless of the outcome of the nomenclature proposal)
**Abstain**

Regards, Ashley

**Reply by joelnitta** [reply]
Hi @DrAshleyField, I'd like to respond here as well.

> How does our PPG II timeline (it is September 2023 now) compare to the IBC timeline (July 2024)?

I think the idea is to finish [Phase I](https://pteridogroup.github.io/guidelines.html#phase-one) (genus and above classification) within a year or so. AFAIK we haven't established a timeline for [Phase II](https://pteridogroup.github.io/guidelines.html#phase-two) (species level) and beyond.

 > Also, would it be possible to include a provisional vote
 
I don't think we should, as it would considerably complicate how to decide the results of the vote. Currently a proposal passes if it garners > 2/3 of the "Yes" votes. It is not clear to me how one would split the "provisional Yes" and "Yes" votes if such a system were used. Also, as Carl mentioned, in the case that a name got accepted by PPG but the nomenclatural proposal it relies on got rejected by IBC, we could just come back to that name and vote to change it. Following the initial release of PPG II, we plan to issue updates on a more frequent basis (at least not wait another 7 years!)

**Comment by GuguYanez** [comment]
Hi! I totally agree with what you have discussed and propose. However, there is something that is not clear to me about the dates planned to finish phase I. Would phase I then end after July 2024? That is, after the IBC? What do we consider "completion," submission for publication?

If this is correct, then perhaps there is an easier way to avoid us having to evaluate and vote now on something that has a chance (however low) of changing after the IBC.

We could do proposals that involve nomenclatural changes but accumulate them so that they can be voted on after the IBC. That is, they could be available to be read by all of us but be reserved for the August 2024 Ballot (probably the last Ballot?). Obviously this option is in the event that the completion date of phase I (if this means submission for publication) is planned after July 2024. If this is not the case, I agree with the option that you initially presented.
I understand that the only problem with this proposal involves having to re-vote already approved proposals (as is the case of Dennstaedtia).

Probably most proposals presented to the nomenclature committee will be approved if they are well founded, but wait for the IBC  (if that is possible) would be a good way to ensure the stability of what we publish. The PPG is the main reference for researchers around the world, especially for those not familiar with the taxonomic discussions of ferns. But also for other actors such as students, teachers and decision makers in the government sphere. While it is great that we are thinking about more frequent updates, I think that what we publish should be as stable as we can achieve.

Happy for this space for debate! :)

**Comment by joelnitta** [comment]
Thanks @GuguYanez for the suggestion. 

I would consider "completion" of Phase I to be the publication of PPG 2.

It is rather by coincidence that our (very rough) projected end for Phase I coincides with IBC (around July 2024). In that sense, what you suggest makes sense. However, we still plan to continue updating PPG **after** IBC XX. In that case, if we delay voting on any proposal that involves a nomenclatural change that needs to be approved by the IBC nomenclature section, we will have to wait another 6 years, which I don't think is feasible.

There is of course a tradeoff between stability and flexibility - as you say, we want the taxonomic system to be stable for a wide range of users. However, we also need to be able to update it to reflect our current understanding of phylogeny, and it is in the best interest of all involved to do so in a timely manner. Given that the risk of having a PPG name be rendered invalid by the nomenclature committee is relatively low in most cases (and we can then update the PPG taxonomy if needed), I think it is appropriate to continue with what we have outlined above (in cases where a taxonomic change requires approval by IBC, as long as a formal nomenclatural proposal has been submitted to IBC, PPG can vote on the matter).

**Comment by Ivan-Valdespino** [comment]
The matters under discussion will be resolved one way or another by decisions taken by the IBC nomenclature section and, thereby, will impact PPGII. Therefore, I believe the suggestion made by @GuguYanez is sensible enough to accommodate nomenclatural stability and the discussion of interesting proposals that are based on phylogenetic analysis. In addition, both the Selaginella and Dennstaedtioideae matters can be adequately pointed out in the context of a final PPG II, just as similar matters were pointed out in PPGI.

Notwithstanding arguments advanced by Carl, I still fail to see the advantages of taking a formal vote now on unsettled issues to be considered by the IBC nomenclature section. Given that @joelnitta has confirmed that there is a plan to continuously update PPG, further nomenclatural changes and adjustments can occur down the line, taking into account the results of the IBC nomenclature section.

I defer to experts working on the Dennstaedtioideae and trust they can provide a sound solution regarding it. However, in the case of the proposal to replace the type species of Selaginella and splitting it into several genera and subdividing the family Selaginellaceae into several subfamilies, I believe this course of action is premature and insufficiently justified. This is evident once one carefully considers the arguments provided to replace the type and split the genus. Current molecular phylogenetic studies on the genus can be interpreted in one way or another, depending on the DNA sequence data and methods employed. This is reflected in the current competing classification schemes proposed. Some characters that define the current type species (i.e., S. selaginoides) are not limited to the narrowly circumscribed, resurrected genus Selaginoides. In fact, some of these characters are also present in the proposed replacement-type species of Selaginella (i.e., S. flabellata) and are also found across other proposed segregate genera. Therefore, it is unlikely that this proposed replacement type could be considered an ideal candidate to support the proposal by Wan et al. (2023).

In addition, proposed segregated genera are, for the most part, taxonomically undiagnosable using morphology. Many of the characters used to define segregated taxa in a dismembering scenario of Selaginella are homoplasious. Likewise, some morphological characters said to define proposed segregated genera are ambiguous and occur across different groups. Furthermore, molecular studies show that some of the analyzed species are not monophyletic, which I suspect is due to taxa misdetermination in some cases. This is a well-known issue that has not been addressed. The matter is still further compounded by the currently limited sampled taxa, which do not adequately represent the geographic distribution and diversity of the genus. As a result, species-rich regions remain largely unstudied. Another aspect to consider is that some species placed within segregated genera lack the morphological characteristics argued to distinguish them, which is most likely a reflection of an inadequate understanding of the taxonomy of the concerned taxa. Additionally, some reinstated species within proposed segregated genera were not included within the molecular phylogenetic study used to support dismembering Selaginella. Yet, these species were previously subsumed under other species names in the genus Selaginella; however, no justification was provided for their resurrection and placement in segregated genera. Further arguments can be provided as support for not agreeing with the proposal by Wan et al. (2023). However, further discussion on the matter must await a formal publication.
